cmd/baton-procore/main.go

```diff
@@ -42,13 +40,13 @@ func main() {
 	}
 }
 
-func getConnector(ctx context.Context, v *viper.Viper) (types.ConnectorServer, error) {
+func getConnector[T field.Configurable](ctx context.Context, config T) (types.ConnectorServer, error) {
```

Comment:
if you update this to `*cfg.Procore` instead of `config T` you should be able to just use `cfg.ClientId` for the value

I updated baton-template recently so this change won't be needed in new connectors

Reply:
Fixed in b539c84307e01a328128ed575f2da817ec2ad1d6